manta-trusted-setup/src/groth16/ceremony/signature.rs

```diff
@@ -140,7 +141,7 @@ where
             verifying_key,
             &Message {
                 nonce,
-                encoded_message: bincode::serialize(message)?,
+                encoded_message: serde_json::to_vec(message)?,
```

Comment:
@BoyuanFeng Why are we signing a json string instead of signing the bytes?? 🤔

Reply:
Yeah, I agree with @tsunrise, lets's go back to `bincode`.

Reply:
1. I kind of feel there is no large difference between using `bincode` or `serde_json` since both `bincode` and `serde_json` use `serde`.
2. We can use one less crate if we only use `serde_json` instead of using both `serde_json` and `bincode`, since we use `serde_json` for all network messages,

Reply:
Also, we shouldn't be using `serde_json` explicitly for network messages since `tide` and `reqwest` can do this automatically for us.